heritability/ldsc_pipeline.py

```diff
@@ -1,18 +1,21 @@
 __author__ = 'Rahul Gupta'
 
-from ukbb_pan_ancestry.heritability.ldsc_ukbb_div_pops_constants import *
-from ukbb_pan_ancestry.resources.results import get_variant_results_path
-from ukbb_pan_ancestry.export_results import get_pheno_id
 import hailtop.batch as hb
 import hail as hl
-hl.init(spark_conf={'spark.hadoop.fs.gs.requester.pays.mode': 'AUTO',
+hl.init(spark_conf={'spark.hadoop.fs.gs.requester.pays.mode': 'CUSTOM',
+                    'spark.hadoop.fs.gs.requester.pays.buckets': 'ukb-diverse-pops-public',
                     'spark.hadoop.fs.gs.requester.pays.project.id': 'ukbb-diversepops-neale'})
 from hail import hadoop_exists, hadoop_ls
 import argparse
-import os, re, math
+import os, re, math, sys
 import pandas as pd
 import numpy as np
 
+sys.path.append('/Users/rahulgupta/Desktop/')
```

Comment:
ಠ_ಠ

Reply:
ugh this is because i had two branches going at once

Reply:
this should be gone